### PR TITLE
Add Docker Desktop Stable 2.2.0.5 release notes

### PR DESCRIPTION
**Pull request by mat007**
### Proposed changes

Adding Docker Desktop Stable 2.2.0.5 release notes.
@usha-mandya 👀 🙏

### CODE REVIEW

File: docker-for-mac/release-notes.md [hunk 1]
```diff
@@ -13,10 +13,19 @@ This page contains information about the new features, improvements, known issue
...
 [What to know before you install](install.md#what-to-know-before-you-install).
 
+## Docker Desktop Community 2.2.0.5
+2020-03-30
+
+### Bug fixes and minor changes
+
+- Removed dangling `/usr/local/bin/docker-machine` symlinks which avoids custom installs of  Docker Machine being accidentally deleted in future upgrades. Note that if you have installed Docker Machine manually, then the install might have followed the symlink and installed Docker Machine in `/Applications/Docker.app`. In this case, you must manually reinstall Docker Machine after installing this version of Docker Desktop. Fixes [docker/for-mac#4208](https://github.com/docker/for-mac/issues/4208).
+
+> [Download](https://hub.docker.com/editions/community/docker-ce-desktop-mac/)
```
Comment (usha-mandya) [inline docker-for-mac/release-notes.md]: Could you move the download link below the release date?

**Comment by usha-mandya** [comment]
Thanks @mat007. Suggested a minor change. Looks good otherwise

File: docker-for-mac/release-notes.md (hunk 1 above)
Comment (mat007) [reply docker-for-mac/release-notes.md]: Oh nice catch, thanks!

**Comment by mat007** [comment]
Done!

File: docker-for-windows/release-notes.md [hunk 2]
```diff
@@ -13,10 +13,22 @@ This page contains information about the new features, improvements, known issue
...
+
+> [Download](https://hub.docker.com/editions/community/docker-ce-desktop-windows/)
+
+### Bug fixes and minor changes
+
+- Docker Desktop generates `fsnotify.WRITE` events in Linux containers when files are changed on the host. Fixes [docker/for-win#5530](https://github.com/docker/for-win/issues/5530#issuecomment-585572414).
+- Fixed a race condition in `readlink` on shared volumes using "mfsymlinks". Fixes [docker/for-win#5793](https://github.com/docker/for-win/issues/5793).
+- Made the VM time sync more reliable. See [docker/for-win#4526](https://github.com/docker/for-win/issues/4526).
+- Fixed bug where files did not change in shared volumes when using `volumes_from` in `docker-compose.yml`. See [docker/for-win#5530](https://github.com/docker/for-win/issues/5530).
+
```
Comment (stephen-turner) [inline docker-for-windows/release-notes.md]: I think we fixed https://github.com/docker/for-win/issues/6016 as well, didn't we?

Comment (mat007) [reply docker-for-windows/release-notes.md]: Yes, nice catch.

Comment (mat007) [reply docker-for-windows/release-notes.md]: Fixed, thanks!

File: docker-for-mac/release-notes.md [hunk 3]
```diff
@@ -13,10 +13,19 @@ This page contains information about the new features, improvements, known issue
 For information about Edge releases, see the [Edge release notes](edge-release-notes). For Docker Desktop system requirements, see
 [What to know before you install](install.md#what-to-know-before-you-install).
 
+## Docker Desktop Community 2.2.0.5
+2020-03-30
```
Comment (usha-mandya) [inline docker-for-mac/release-notes.md:17]: @mat007 We need to update the release date.

Comment (mat007) [reply docker-for-mac/release-notes.md:17]: Ah yes, doing it!

Comment (mat007) [reply docker-for-mac/release-notes.md]: Done.

**Pull request merged**